### PR TITLE
[Audio-3.2] Fix an issue with mixplaylist being recognised as single tracks

### PR DESCRIPTION
**Pull request by Drapersniper**
Signed-off-by: Drapersniper <27962761+drapersniper@users.noreply.github.com>

### Type

- [x] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes

This fixes some mix playlists being recognised as single tracks

### CODE REVIEW

File: redbot/cogs/audio/dataclasses.py [hunk 1]
```diff
@@ -381,14 +381,17 @@ def _parse(track, **kwargs):
...
-                            returning["single"] = True if _has_index else False
+                            returning["playlist"] = not _has_index
+                            returning["single"] = _has_index
+                        elif any(k in track for k in ["list="]):
+                            returning["track_index"] = 0
+                            returning["playlist"] = True
+                            returning["single"] = False
                         else:
                             returning["single"] = True
                     elif url_domain == "spotify.com":
```
Comment (AHIRAISSHY) [inline redbot/cogs/audio/dataclasses.py]: yesh

**Review by AHIRAISSHY — COMMENTED** [review]
yesh

File: redbot/cogs/audio/dataclasses.py (hunk 1 above)
Comment (Vexed01) [reply redbot/cogs/audio/dataclasses.py]: # 🤔

**Review by Kowlin — APPROVED** [review]

**Comment by Drapersniper** [comment]
Conflicts resolved ready to be merged

**Pull request merged**